src/Microsoft.DotNet.Cli.Utils/FileNameSuffixes.cs

```diff
@@ -62,6 +64,14 @@ public static PlatformFileNameSuffixes CurrentPlatform
             StaticLib = ".a"
         };
 
+        public static PlatformFileNameSuffixes FreeBSD { get; } = new PlatformFileNameSuffixes
```

Comment:
Can't we just use the linux one above for this? Why do we need a specific and duplicated one?

Reply:
+1, just `public static PlatformFileNameSuffixes FreeBSD { get; } = Linux;`
Then by same logic `public static PlatformFileNameSuffixes Windows { get; } = DotNet`?